example/driver/simpledriver.go

```diff
@@ -46,14 +83,24 @@ func (s *SimpleDriver) HandleReadCommands(deviceName string, protocols map[strin
 		err = fmt.Errorf("SimpleDriver.HandleReadCommands; too many command requests; only one supported")
 		return
 	}
-
+	// reqs[0].RO.Object == "onvif_profiles"
```

Comment:
Added line but commented out. Can this be removed?

Reply:
Updated with new commit.